app.ts

```diff
@@ -0,0 +1,414 @@
+// Get the container from the html where the data will be stored
+let recordNav: any = document.querySelector("#record-navigation-container"); //Navigation area;
+let headingColumns: any = document.querySelector("#column-headings-container"); //Headings;
+let infoColumns: any = document.querySelector("#info-columns-container"); //Information;
+let count = 1;
+let debounce = (func: any, delay: any) => {
+  let timer: any;
+  return function () {
+    clearTimeout(timer);
+    count++;
+
+    timer = setTimeout(() => {
+      func();
+      count = 1;
+    }, delay);
+  };
+};
+
+// Creates the navigation.
+function createNavigation() {
+  recordNav.innerHTML = `
+  <div class="navigation-btns">
+    <button class="first-page-btn">First Page</button>  
+    <button class="previous-records-btn">Previous</button>
+    <button class="next-records-btn">Next</button>
+    <button class="last-page-btn">Last Page</button>  
+    <button onclick="recordSelection()" id="confirmation-btn">Get Record</button>
+  </div>
+  <div class="current-page-container">
+    <p class=current-page></p>
+  </div>
+
+  `;
+}
+
+// The functionality to move through an amount of records.
+function nextPrevious(firstId: any) {
```

Comment:
Use the `any` type sparingly. Go through the JavaScript Style Guide to see preferences and standards.